frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.tsx

```diff
@@ -134,7 +134,6 @@ export class ClickActionsPopover extends Component<
         opened={!!popoverAnchor}
         onClose={this.close}
         position="bottom-start"
-        width={700}
```

Comment:
This change does not affect the Core App, because the custom `use-popover` override is used there.